sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/DigitalTwinsRestClient.cs

```diff
@@ -19,6 +19,234 @@ internal partial class DigitalTwinsRestClient
     {
         private const string DateTimeOffsetFormat = "MM/dd/yy H:mm:ss zzz";
 
+        /// <summary>
```

Comment:
A little confused why these need to be added, seems like its related to other parts of the SDK not bulk right?

Reply:
In other words - still a little confused about what changed in bulk to cause this - or is this something we should / could do as a separate PR? would it need more reviewing than bulk?

Reply:
I _think_ this was caused by the additionalProperties that were added, which are generic dictionaries. I.e., the response type really changed from a generic object to a dictionary. In our SDKs we previously already accounted for this case to some extent, but additional changes are necessary with the updated response model. I think in the solutions client we have something similar.